Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
@thinkyhead @Roxy-3DPrintBoard What is the issue you are trying to resolve right now?  Did that patch to fix the contrast issue I submitted ever get merged?  I'll review this thread tonight after work.